### DISCUSSION TITLE
[A-L] (16 Gennaio 2023) Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#368] [Esami A-L]
Consideriamo la seguente relazione $\Large \prec$ definita su coppie di intervalli chiusi della retta reale:

<div align="center">

$\Large [x, y] \prec [w, z]$ se e solo se $\Large [x, y] = [w, z]$ oppure $\Large y \leq z$.

</div>

Indicare se le seguenti affermazioni sono vere o false.

1. La relazione $\Large \prec$ è riflessiva.

2. La relazione $\Large \prec$ è simmetrica.

3. La relazione $\Large \prec$ non è transitiva.

**Comment by cek753** [comment]
1- **Vera**  Ogni elemento può essere in relazione con se stesso per la definizione data.

2-**Falso**  Se [x,y]=[w,z] allora vale la simmetria perchè [w,z]=[x,y]. Ma nel caso in cui y ≤ z non è sempre detto che soddisfi la simmetria.
Esempio: [4,5]≺[6,10] ma [6,10] non è in relazione con [4,5].

3-**Falso**
